sopel/modules/url.py

```diff
@@ -186,16 +186,12 @@ def find_title(url, verify=True):
     """Return the title for the given URL."""
     response = requests.get(url, stream=True, verify=verify)
     try:
```

Comment:
I think `with requests.get(url, stream=True, verify=verify) as response:` is appropriate, seeing that py2.7 is required